### PR TITLE
Saul/select SubCommand Unit Test methods

### PR DESCRIPTION
**Pull request by eliosfg**

### CODE REVIEW

File: src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandSelectTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.fundacionjala.at15.pokemon.io.Writer;
+import org.junit.Test;
+import picocli.CommandLine;
+
+public class SubCommandSelectTest {
+    @Test
+    public void itShouldSelectAPokemon() {
+        Pokemon newPokemon = new Pokemon(200, "Pikachu");
+        Writer.writeToJson(newPokemon);
+
```
Comment (adhes14) [inline src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandSelectTest.java:23]: Try create a current json file before using select command:
`Writer.writeToJson(new CurrentEntities());`

**Review by adhes14 — COMMENTED** [review]
You need current json file

**Review by lechemon4000 — APPROVED** [review]
Good job! this will improve the code coverage

**Review by adhes14 — APPROVED** [review]
Well done!

**Pull request merged**